spec/support/acceptance_helper.rb

```diff
@@ -0,0 +1,19 @@
+module AcceptanceHelper
+
+  def sign_in(user)
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_on 'Log in'
+  end
+
+  def fill_passwords
+    fill_in 'Password', with: '12345678'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.